redpanda/values.yaml

```diff
@@ -163,9 +161,108 @@ listeners:
         cert: schema
         requireClientAuth: false
 #
-# Log level
-# Valid values are info, warn, debug, trace
-logLevel: info
+# Logging
+logging:
+  # Log level
+  # Valid values (from least to most logging) are warn, info, debug, trace
+  logLevel: info
+  #
+  # Send usage stats back to Redpanda
+  # See https://docs.redpanda.com/docs/cluster-administration/monitoring/#stats-reporting
+  usageStats:
+    # rpk.enable_usage_stats
+    enabled: true
+    # Your organization name (optional)
+    organization: jlp-test
+    # Your cluster ID (optional)
+    clusterId: helm-cluster-01
+#
+resources:
+  # Both Redpanda and Kubernetes have multiple ways to allocate resources.
+  # There are also several associated parameters that impact how these resources are used by
+  # Kubernetes, the Redpanda app, and the subsystem Redpanda is built on (Seastar).
+  # This section attempts to simplify allocating resources by providing a single location
+  # where resources are defined.
+  # Helm sets these resource values within the following templates:
+  # - statefulset.yaml
+  # - configmap.yaml
+  #
+  # The default values below are what should work for a development environment.
+  # Production-level values and other considerations are provided in comments
+  # if those values are different from the default.
+  #
+  cpu:
+    # Redpanda makes use of a thread per core model described here:
+    # https://redpanda.com/blog/tpc-buffers
+    # For this reason, Redpanda should only be given full cores (cores parameter below).
+    #
+    # NOTE: You can increase cores, but decreasing cores is not currently supported:
+    # https://github.com/redpanda-data/redpanda/issues/350
+    #
+    # Equivalent to: --smp, resources.requests.cpu, and resources.limits.cpu
+    # For production: 4 or greater
+    cores: 1
+    #
+    # Overprovisioned means Redpanda won't assume it has all of the provisioned CPU.
+    # This should be true unless the container has CPU affinity (eg. min and max above are equal).
+    # Equivalent to: --idle-poll-time-us 0 --thread-affinity 0 --poll-aio 0
+    # overprovisioned: false
+  #
+  memory:
+    # Enables memory locking.
+    # For production: true
+    # enable_memory_locking: false
```

Comment:
I would have liked to provided more details on what this `enable_memory_locking` parameter does.